File: app/src/main/java/com/raminarmanfar/healthapp/HealthDBContract.java
package com.raminarmanfar.healthapp;

import android.provider.BaseColumns;

public class HealthDBContract {

    private HealthDBContract () {}

    public static final class CaloriesCategories implements BaseColumns {
        public static final  String TABLE_NAME = "caloriesCategories";
        public static final  String COL_NAME = "name";
        public static final  String COL_TYPE = "type";
        public static final  String COL_DESC = "description";
        public static final  String COL_TIMESTAMP = "timestamp";
    }
}

File: app/src/main/java/com/raminarmanfar/healthapp/HealthDBContract.kt
package com.raminarmanfar.healthapp

import android.provider.BaseColumns

class HealthDBContract private constructor() {
    object CaloriesCategories : BaseColumns {
        const val TABLE_NAME = "caloriesCategories"
        const val COL_NAME = "name"
        const val COL_TYPE = "type"
        const val COL_DESC = "description"
        const val COL_TIMESTAMP = "timestamp"
    }
}